Pirate Campaign Prologue: Three missions of smuggling, slavery, and swashbuckling. (#6298)
endless-sky/endless-sky: data/human/pirates/pirate intro.txt

			choice
				`	(Peruse a few pages, perhaps you'll learn something.)`
				`	(Nope. If I wanted to read things I would've gotten an honest job.)`
					decline
			`October 3, '94`
			`	Bastards took the farm. Funny how you go your whole life never seeing a man in a suit, and the first one you do is there to ruin everything. Well, fine. I'm strong. I'm smart. I can go to the city and find work.`
			`	The next several pages are filled with landscape sketches, scenes from a verdant farming world. Trees loom tall in the divisions between fields, flowers are abundant where nature hasn't been tamed by the colonist's hand. Here, a flock of sheep graze. There, a ship streaks across the sky, the rest of the drawing seemingly contorted around it as if the world itself is focusing on the vessel as it speeds into the unknown.`
			`October 28, '94`
			`	Empty stomach, empty pockets, no jobs to be found. Not a broom that needs pushing, and the mob at the factory wanting employment is at least five men deep and I dare not count how wide.`

> 			`	Folk, man and woman, young and old, sit idle in the streets of a modestly sized town. Long faces, hollow with hunger, impassive in idleness, turn their gazes towards the one drawing them. There's the occasional drawing of a field or burst of beauty as captured in the pages before, but the overall spirit has shifted to the sullen.`

Author: @Pointedstick I've shoved all the non-narration within quotation marks, will that serve to disambiguate for the time being? It's not ideal in my view, but likely the best we can do until we get unicode support.

> 			`	Folk, man and woman, young and old, sit idle in the streets of a modestly sized town. Long faces, hollow with hunger, impassive in idleness, turn their gazes towards the one drawing them. There's the occasional drawing of a field or burst of beauty as captured in the pages before, but the overall spirit has shifted to the sullen.`

Reviewer: Yeah, that should be fine.

	choice
		`	"Looks good to me, thank you."`
			goto complete
		`	"What in the void is this?"`
	`	The dark-haired officer simply smiles gently. "It is what it is. Now take it and run along to your employer. We're done here."`
	branch complete
		not "clobbered"
	`	The soldiers give you a moment to dress yourself and gather your things back up, though the surly looks you're fixed with make it clear they're keen to see you go.`
	label complete
	action
		payment -1000000
	`	The transaction is complete. You've spent a million credits out of your own pocket for something that needs to be delivered to a mysterious mercenary you met in a stinking bar. This is definitely the path of wisdom that you walk. Thankfully, no further obstacles ambush you as you return to your ship.`
		accept
	label retreat
	`	You make your apologies - wrong hangar - and beat a hasty retreat. The door seals behind you. On the way to your ship you pass the janitor once more, his ear firmly pressed to the handle of his mop. Upon seeing you he becomes visibly startled and drops his tools all over the floor. What a strange fellow.`
		decline
	

> 

Reviewer: :+1:

			choice
				`	"What's going on here?"`
				`	(Stroll into the middle of town. Lock eyes with the leader of the slavers. Assume a gunfighter's stance.)`
					goto duel
				`	(Return to your ship. This is none of your business.)`
					goto retreat
				`	(Return to your ship... and start strafing the slavers.)`
					goto attack
			`	"No business of yours, friend. We know who you are, and we thank you for running off the last band of lowlifes that laid claim to these bodies. Now, if you'll excuse, we've got work to do. Piss off." The slaver chief roundly ignores any further attempts to communicate, but he at least seems to be in a good enough mood to not have you shot on the spot.`
			choice
				`	"These people are my responsibility, and I'll fight you for their freedom."`
				`	(Return to your ship. This is none of your business.)`
					goto retreat

> 				`	(Return to your ship... and start strafing the slavers.)`
> 					goto attack	

Reviewer 2: ```suggestion
				`	(Return to your ship... and start strafing the slavers.)`
					to display
						"armament deterrence" > 0
					goto attack
```

Reviewer 2: That will prevent the player from "strafing the slavers" if they have to ships with weapons.

			choice
				`	(Draw!)`
			`	The slaver boss is fast, but you're faster. You're able to slip your sidearm out of its holster, level it, and cleanly blow him away before he can so much as draw a bead on you - and so it's with the thrill of victory that you die, as you're unceremoniously cut down by massed fire from the gang of slavers surrounding your little duel.`
			`	The few colonists that are left behind after the slavers have filled their holds honor you with a short burial and a bottle of local moonshine poured out over your cairn in respect for the brave, but fairly ill-conceived, attempt to defend them.`
				die
			label attack
			`	You retreat to your ship, unable to help these poor souls...`
			`	... Or at least, that's what you want the slavers to think. They're caught unaware as you streak back towards the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to so much greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village back to the landing pad.`

> 			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`
> 				launch

Reviewer 2: In case the player is unarmed.

```suggestion
			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`
				to display
					"armament deterrence" > 0
				launch
			`	Survivors pour through the hatches and into their ships. You have no choice but to retreat in your unarmed ship, hoping you can outrun them before they shoot you down.`
				launch
```

			`	As it turns out, they're not there to render tribute unto you as you might have thought. No, the villagers are all being held at gunpoint by a gang of toughs in the center of the village green while a nasty-looking piece of work examines them one at a time, then directs them into one queue or another. Colonists in one are shackled together. Those in the other are made to lie flat with their hands behind their heads. Not a single colonist dares to look up; the spirit of defeat hangs too heavily over these broken souls.`
			choice
				`	"What's going on here?"`
				`	(Stroll into the middle of town. Lock eyes with the leader of the slavers. Assume a gunfighter's stance.)`
					goto duel
				`	(Return to your ship. This is none of your business.)`
					goto retreat
				`	(Return to your ship... and start strafing the slavers.)`
					goto attack
			`	You're given an appraising look-over by the slaver chief. "... Yeah, you'd be too much trouble to sell. What'cha want, kid? We're busy here."`
			choice
				`	"These colonists are my property by right of conquest. If you're taking them, I expect to be fairly compensated."`
				`	"Actually, I was just leaving."`
					goto retreat

> 				`	"Whoah, whoah, whoah, I'm cool with forcing people to give me stuff because I'm holding them at gunpoint, but forcing them into slavery? That's sick."`

Reviewer 3: ```suggestion
				`	"Woah, woah, woah, I'm cool with forcing people to give me stuff because I'm holding them at gunpoint, but forcing them into slavery? That's sick."`
```

	`	"Aye, it be not enough to simply cover yer' ship with the old black-and-red. A pirate must earn the respect - or fear - of their fellows, lest they be seen as fair plunder," the Corsair contributes. This is followed up with an entirely gratuitous but quite hearty "Arrrrgh."`
	`	The Capo speaks last, jaw motionless; his voice is provided by a cybernetic vocoder, mellifluous if gently synthetic. "Indeed. There's three things you'll need to do before you'll be taken seriously. Speaking personally, I'm always on the eye for upcoming talent. Prove that you've got what it takes and I'm sure at least one of us would make it known within our circles that you aren't to be trifled with - and perhaps even worth offering work to. But first, those tasks."`
	choice
		`	(Nod towards the Corsair.) "Does he always speak like that?"`
		`	"Oh, I know! I need to master theft, swordplay, and treasure hunting!"`
			goto hahareferencegoeshere
		`	"Fine. What should I do if I want other pirates to take me seriously?"`
			goto tasks
		`	(This was a terrible idea. Back to the ship.)`
			goto retreat

> 	`	"Oh, leave him be," the Mercenary scolds you. "He always wanted to retire to Buccaneer Bay and, well, you know how they've got that romanticized view of the old Earth tales, back when ships didn't need to be pressurized cans." The Corsair smiles, somewhat sheepishly. He doesn't seem to have the heart to "Aaargh" this time.`

Author: > I assumed that 'they' referred to the people on Buccaneer Bay.

Correct

		`	"Tell me what I need to do."`
			goto explanation
	`	The Capo groans, shakes his head mournfully, and leans back to excuse himself from the conversation for a minute. The Mercenary quickly fills the silence. "Look, it's simple. You shoot people, they shoot back. You knuckle under at every opportunity, they keep shooting because they think you're weak. If you want to be left alone you need to prove that you're not worth the effort it would take to plunder you, yeah? That's... Well, it's what these little tasks we've mentioned are about. You need to establish a reputation. This is how you do it - and once you've done that, other pirates should leave you alone owing to either fear or respect."`
	`	She leans back in her chair, matching gaze with the Capo. "And you, don't push it. Not here, you know what my boss thinks." Wistfulness drips from her words, and her gaze turns past the table and off into the distance. This person's thoughts clearly lie elsewhere at the moment.`
	choice
		`	"I understand."`
		`	"Yeah, I have no idea what you're talking about, but I don't care enough to ask again. Where's the part where I get to be a pirate?"`
	label explanation

> 	`	"Now," the Corsair begins, "I know a juicy prize is going to be sailing-" The other two turn to fix the Corsair with a skeptical gaze. The Capo's maw slips wide for a moment, as if he wants to interrupt, but eventually it closes silently once more, and the Corsair continues. "-SAILING, I say, through Umbral. In fact, any buccaneer worth their salt knows she'll be there, so you're liable to have more than a little competition. Board her, plunder her, keep what ye like - but should ye find an antique wood lockbox, yer' to bring it back to me without peeking within. That'll be my price for spreading the word of yer' glorious ascent to piratehood.`

Author: It's a discreet place to transfer cargo between ships without all the pesky paperwork and being-noticed involved with docking at a planet.

	`	"As for me?" The Devil's Due member takes her turn to speak. "Take a million credits to Smuggler's Den in Men. I don't care where you get the money -in fact, I don't want to know. Just drop by with that million, complete a little transaction I'll have arranged by then, and I'll get you what you want. Prestige. Respect. Fame. Infamy, whatever." A winning smile is flashed your way, and your hand is plucked up, shaken; a datachip is pressed into it. "Contact for an associate of mine, on the station I mentioned. Take the money there, bring the item back to me."`

> 	`	Finally, the Capo takes his turn. "Have you heard of the planet Darkstone, in the system Han? Pathetic bunch of dirt-farmers, only worthy of enslavement. Impose yourself as their master, take the meager tribute they offer, and return. Know that you'll make enemies this way, but really - if you want to be a pirate, I'm sure that's not something that concerns you overmuch."`

Reviewer 2: The tribute battle in Hai Reveal is a bandaid. The intent was that the player couldn't land until defeating the planetary defense fleets. Currently, the only way to do that is to have a tribute battle. Getting tribute from Elenchus causes major lore problems. We had to bend over backwards to come up with a reasonable story justification for it.

In this case, all that is needed is a fleet to defeat. Then you land and threaten to do it again unless they pay you.

> 	`	Finally, the Capo takes his turn. "Have you heard of the planet Darkstone, in the system Han? Pathetic bunch of dirt-farmers, only worthy of enslavement. Impose yourself as their master, take the meager tribute they offer, and return. Know that you'll make enemies this way, but really - if you want to be a pirate, I'm sure that's not something that concerns you overmuch."`

Author: This will be reworked to remove reliance on tributing.

	`	The assembled outlaws have, by this point, exhausted what attention they'd budgeted on your behalf. The Mercenary dismisses you with a wave of the hand; the other two appear more concerned with an argument of their own until the Corsair finally pipes up with a final note for your departure. "Whichever path you take, we expect to be on Greenrock for the foreseeable future. Drop by once you've got something one of us will find impressive; or all, if that's your style."`
		accept
	label retreat
	`	As your nostrils fill with the scent of stale piss, cheap alcohol, and a modest breeze carrying the metallic tang of blood upon it, you come to your senses and realize that perhaps the pirate's life isn't as romantic as you'd made it out to be.`

> 	`	Leering eyes bore into your back as you beat a hasty retreat to your ship. Improbably, the alleys you dart through look even grungier on the route back than they did on the way here.`

Author: Keeping this one

		not "hero of Darkstone"
	#the player claimed Darkstone as a tributary but never visited their new subjects, or took no action upon finding slavers abducting colonists.
	branch conqueror
		and
			not "hero of Darkstone"
			not "slaver of Darkstone"
	#the player claimed Darkstone as a tributary and defended the colonists against a slaver raid.
	branch defender
		or
			has "hero of Darkstone"
			has "defender of Darkstone"
	#the player sold the colonists of Darkstone to slavers
	branch slaver
		has "slaver of Darkstone"
	label conqueror
	choice
		`	(Address the Capo.) "I conquered Darkstone, defeating the gang that once claimed it. The inhabitants now offer me tribute."`
	`	The Capo nods in approval. "And a mighty fine job you did, too. Without those ruffians my boys were able to ply their trade far more easily. You've my thanks - and, as I promised, I've got some introductions to make on your behalf.`
	choice
		`	"What trade is that, exactly?"`
		`	"Thank you. I've conquered a world, but I'm only getting started."`
			goto outlawapproval
	`	"Human resources, naturally." The Capo lets out a dark laugh, eyes thin and aspect sharklike as he fixes you with a predatory grin.`
	choice

> 		`	(Remain impassive, not caring one way or another about the fates of others than yourself - slavery is neither your business nor your problem.)`

Reviewer 2: I think the story would have more character if the player could say something to convey these ideas, rather than making faces.

	#the ring token is for integration into the pirate campaign
		#goto ring
		goto outlawapproval
	label slaveryspeech
	`	"Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that. Pirates who think they can survive - or more amusingly, thrive - and still hold onto their precious morals; they're not long for this galaxy."`
	`	The Corsair's lips curl into a decidedly displeased scowl, but he drowns any words in liquor before they might threaten to spill forth.`
	`	"Now. Don't be a fool, and remember my words." The Capo relaxes, stretching back in his seat. He's regarding you coolly, contemplatively. "But I'm a man of honor; you did as you were instructed, and so I'll fulfill my end of the bargain."`
		goto tributeskip
	label defender
	choice

> 		`	(You address the Capo.) "I conquered Darkstone, defeating the gang that once claimed it. When others came to enslave my subjects, I defeated them as well."`

Reviewer 2: You could indicate the target of the sentence by something like "I conquered Darkstone, Copo, defeating the..."

	`	"You little shit. I gave you a job to do, an opportunity, and you repay me by killing my men? Did you think you could just muscle in to the trade without people backing you up?" Cold rage burns within the cyborg's eyes, his expression hard to read given lack of lips and most facial muscles.`
	choice

> 		`	"They got what was coming to them. That territory was mine by right of conquest, and by trespassing they signed their own death warrants. You respect my turf and I'll respect you, understood?"`

Reviewer 2: Actually, I'd split that into two choices, for color:

- "They got what was coming to them. You respect my turf and I'll respect you, understood?"
- "That territory was mine by right of conquest, and by trespassing they signed their own death warrants."

You choose between a direct threat and an implied threat.

			goto grudgingrespect
		`	"I didn't know those were your men, I simply saw raiders coming in to steal from my new tributaries and I didn't want that to happen. I'm sorry."`
			goto weak

> 		`	"Damn slavers. If I'd known that's what you'd intended all along I never would have gone along with your plan. You disgust me."`

Reviewer 2: Why would you remove that? The player has entered the dark side of the Endless Sky. Disgusting retorts are part of that culture's charm.

	label outlawapproval
	`	"Now, I'll do as I said and help make your name known to those who matter, and the lowlifes that don't. Pirates will respect you for the deeds you've done."`
		goto tributeskip
	#label ring
	#`	The Capo extends his palm, a shining silver ring glistening in the center. Unadorned by any manner of stone, it's a simple, subdued piece of jewelry. Upon the front a crest has been stamped. "A token of my approval. Consider it sponsorship, and my trust that you'll do what need be done. Take it to Gienah. People will know what it means, and you'll find lucrative work that lines up nicely with the skillset you've demonstrated."`
	#`	The cyborg offers a genial smile - or at least as much as one can without lips. It's uncanny how expressive articulated artificial jaws can be these days.`
	label tributeskip
	branch boardingskip
		not "Pirate Intro: Boarding: done"
	choice	

> 		`	(Slap the top of the Corsair's trunk.) "I've brought the antique lockbox and defeated the other pirates who came for it."`

Reviewer 2: The player's acts of violence are part of the story. Just because other campaigns have Captain Last be a soulless chatbot doesn't mean the Pirate Campaign has to.

	choice
		`	"Looks good to me, thank you."`
			goto complete
		`	"What in the void is this?"`
	`	The dark-haired officer simply smiles gently. "It is what it is. Now take it and run along to your employer. We're done here."`
	branch complete
		not "clobbered"
	`	The soldiers give you a moment to dress yourself and gather your things back up, though the surly looks you're fixed with make it clear they're keen to see you go.`
	label complete
	action
		payment -1000000
	`	The transaction is complete. You've spent a million credits out of your own pocket for something that needs to be delivered to a mysterious mercenary you met in a stinking bar. This is definitely the path of wisdom that you walk. Thankfully, no further obstacles ambush you as you return to your ship.`
		accept
	label retreat

> 	`	You make your apologies - wrong hangar - and beat a hasty retreat. The door seals behind you. On the way to your ship you pass the janitor once more, his ear firmly pressed to the handle of his mop. Upon seeing you he becomes visibly startled and drops his tools all over the floor. What a strange fellow.`

Reviewer 3: https://dictionary.cambridge.org/dictionary/english/beat-a-retreat
Cambridge Dictionary considers it valid.

> 

Reviewer 4: You should probably put a copyright header here.

Reviewer 2: No, not a copyright header. Some people make the moral choice to release a work to the public domain. As this is the pirate campaign, releasing without copyright fits the style of the campaign. It is also a valid choice by the author, if they want to do it. However, the lack of a header does *not* release it without copyright.

It needs a CC0 public domain dedication, and the style checker has to be updated to accept CC0. I'll do a PR to this branch shortly to make that update.

You can't *force* someone to claim copyright, nor would it be morally right to do so.

Reviewer 5: @UnorderedSigh I'd let Lake speak for himself before typing out a debate on having copyright or not for the game. It's common practice for all of our content writers. It's okay if you don't like it, but don't do the opposite and try to push someone *away* from doing so. Leave it grey-lined.

Reviewer 5: It appears Lake on discord is going to decline copyright. That is totally fine. In the future Sigh, please do not make these decisions for content that isn't yours, and do not press them in a direction of your own choice. Please let the creator decide for themself.

Reviewer 2: I haven't spoken for anyone. I've just given the suggestion. However, both my sister and I have asked to release in CC0 and been denied, so I'm happy to fight for a fellow pirate.

I made PR to this branch which adds a CC0 dedication to the `pirate intro.txt` with an appropriate header line.

https://github.com/Endless-Sky-Pirate-Campaign/endless-sky/pull/27

Reviewer 6: The whole game is MZ unless specified otherwise anyway, isnt it?

Reviewer 2: The file, on its own, is only (C) MZ if there is a notice in the file saying so.

Author: Thanks, Sigh!

mission "You Are A Pirate"
	landing
	invisible
	source
		government "Pirate"
	to offer
		not "chosen sides"
		or	
			and
				or
					"reputation: Republic" < -10
					"reputation: Syndicate" < -10
					"reputation: Free Worlds" < -10
				"reputation: Merchant" < -10
			"reputation: Pirate" > 20
	on offer
		conversation "You Are A Pirate"
	on accept
		event "became a pirate"
		fail

conversation "You Are A Pirate"
	`Preparing to land is second nature by now: patch any perforations in the inner hull, get in touch with ground control, and then make sure your chosen parking pad isn't controlled by extortionists. When you disembark, it's with a pistol at your hip, bribe money in your pocket, and a swagger in your step; to leave behind any one of these is to invite trouble. Perhaps a life of violence on the high skies and association with the rougher elements of the galaxy weren't what you had in mind when you left your homeworld, but it's what you found.`
	choice
		`	(This is my life now. Piracy, smuggling, the existence of the outlaw. I'm ready to embrace it.)`
		`	(This isn't what I wanted. This isn't the life I imagined when I decided to take to the high skies.)`
			decline

> 	`	Resolve will carry you far in this world. Tonight it's carried you to a spaceport paint supplier; you return to your vessel with a pallet loaded with blacks and reds. Apply it to your vessel, make your intentions known to the galaxy. There can be no pretense of peace when you fly girded in blood and midnight, but maybe that's exactly the way you want it to be.`

Author: It does now.

		`	(No. Not now, perhaps not ever. The galaxy at large may consider you a pirate, but you're not ready to adopt their colors.)`
			decline
	`	Some paint covers all manner of sin. Some speaks of the intention to commit yet more. You've applied the latter sort to your ship.`
		accept

mission "I Want To Be A Pirate"
	clearance
	name "Become a pirate"
	description "Being a pirate in the eyes of the law isn't the same as being one in the eyes of fellow buccaneers. You'll have them know your name - but first you must make one for yourself."
	source
		government "Pirate"
	destination "Greenrock"
	to offer
		not "You Are A Pirate: offered"
		not "chosen sides"
		or
			"combat rating" > 150
			has "ships: Interceptor"
			has "ships: Drone"
			has "ships: Fighter"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
	on offer
		conversation "I Want To Be A Pirate"
	to complete
		or
			has "Pirate Intro: Boarding: done"
			has "Pirate Intro: Wealth: done"
			has "Pirate Intro: Domination: done"
	on complete
		event "became a pirate"
		conversation "I Want To Be A Pirate Conclusion"

conversation "I Want To Be A Pirate"

> 	`The drinking holes flanking this outlaw spaceport are a motley mix indeed, each apparently shabbier, dingier, or more raucous than the last. Perhaps that's just the way you like your establishments; perhaps not. Whatever the case, a sense of purpose has brought you here, far from the factories and fields of New Boston. Out here to the fringes of society.`

Reviewer 7: We've currently got a handful of missions that also make explicit reference to you being from New Boston. Once we have the actual starts in and codify what those conditions look like, I'm sure they'll all be updated at once.

> 	`The drinking holes flanking this outlaw spaceport are a motley mix indeed, each apparently shabbier, dingier, or more raucous than the last. Perhaps that's just the way you like your establishments; perhaps not. Whatever the case, a sense of purpose has brought you here, far from the factories and fields of New Boston. Out here to the fringes of society.`

Author: Yup. Once we have alt starts it won't be too much trouble to just search the codebase for "New Boston" and make a list of what needs to be edited, I figure. So this should work fine for now.

	`	Instinct - or an overabundance of swashbuckling serials consumed during your childhood - brings you to the back of a smoky bar, where a trio of grizzled ruffians sit: an eyepatched Corsair festooned in somewhat anachronistic couture; a jewelry-encrusted Capo with a mechanical jaw and the haughty bearing of a born and bred gangster; and a Mercenary wearing a beret and military fatigues. A patch affixed to her shirt identifies her as a member of the "Devil's Due Contracting Company". The three turn to stare at you. You came here for a reason, didn't you?`
	choice

> 		`	"My name is <first> <last>, and I want to be a pirate."`

Author: I like that. Both will lead to the same outcome, but I can definitely add in another dialogue choice. This one was intended as direct homage to The Secret of Monkey Island.

	#the ring token is for integration into the pirate campaign
		#goto ring
		goto outlawapproval
	label slaveryspeech
	`	"Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that. Pirates who think they can survive - or more amusingly, thrive - and still hold onto their precious morals; they're not long for this galaxy."`
	`	The Corsair's lips curl into a decidedly displeased scowl, but he drowns any words in liquor before they might threaten to spill forth.`
	`	"Now. Don't be a fool, and remember my words." The Capo relaxes, stretching back in his seat. He's regarding you coolly, contemplatively. "But I'm a man of honor; you did as you were instructed, and so I'll fulfill my end of the bargain."`
		goto tributeskip
	label defender
	choice
		`	(You address the Capo.) "I conquered Darkstone, defeating the gang that once claimed it. When others came to enslave my subjects, I defeated them as well."`

> 	`	"You little shit. I gave you a job to do, an opportunity, and you repay me by killing my men? Did you think you could just muscle in to the trade without people backing you up?" Cold rage burns within the cyborg's eyes, his expression hard to read given lack of lips and most facial muscles.`

Reviewer 8: Wouldn't this be better?
> Cold rage burns within the cyborg's eyes, his expression hard to read given a distinct lack of lips and facial muscles.

> 	`	"You little shit. I gave you a job to do, an opportunity, and you repay me by killing my men? Did you think you could just muscle in to the trade without people backing you up?" Cold rage burns within the cyborg's eyes, his expression hard to read given lack of lips and most facial muscles.`

Author: I'll take JE's edit, that flows much better.

	choice

> 		`	"They got what was coming to them. That territory was mine by right of conquest, and by trespassing they signed their own death warrants. You respect my turf and I'll respect you, understood?"`

Author: This is as it should be.

			`	Completing any of these truly important quests and reporting back to the ne'er-do-wells on Greenrock will make you a pirate.`
				decline

mission "Pirate Intro: Boarding"
	landing
	name "Board a merchant"
	description `Board a merchant.`
	destination "Greenrock"
	waypoint "Umbral"
	to offer
		has "I Want To Be A Pirate: active"
	to fail
		has "I Want To Be A Pirate: done"
	on offer
		event "add merchant targets"
	to complete
		has "Plunder Defense: done"

fleet "Merchant Targets"
	government "Merchant (Target)"
	names "civilian"
	cargo 1
	personality
		heroic
	variant 1
		"Freighter"

government "Merchant (Target)"
	swizzle 5
	
	"player reputation" 10
	"attitude toward"
		"Pirate" -.2
		"Korath" -.2
	"bribe" .05
	"fine" 0
	"friendly hail" "friendly civilian"
	"hostile hail" "hostile civilian"

mission "Plunder Defense"
	name "Return with spoils"
	description `Defeat your rival pirates in Umbral, then return to <destination> with the spoils from the <origin> to confirm your loyalty.`
	boarding
	source
		government "Merchant (Target)"
	to offer
		has "Pirate Intro: Boarding: active"
	on offer
		conversation

> 			`You have successfully boarded the merchant ship, and right on cue, your combat suit's communicator alerts you to your flagship's proximity alert being triggered - evidently several competitors have arrived. Just as the Corsair promised, an aging chest lies among the cargo. You barely have enough time to secure it aboard your ship before the radar contacts get too close for comfort. Defend yourself!`

Reviewer 9: Navy used battlesuit or something during raid on Poisonwood iirc

> 			`You have successfully boarded the merchant ship, and right on cue, your combat suit's communicator alerts you to your flagship's proximity alert being triggered - evidently several competitors have arrived. Just as the Corsair promised, an aging chest lies among the cargo. You barely have enough time to secure it aboard your ship before the radar contacts get too close for comfort. Defend yourself!`

Author: I haven't seen them mentioned, no - but no reason to let potential worldbuilding languish. Being a cool pirate means you either need a cool coat *or* cool armor. Or an armored coat, I suppose.

			choice
				`	(Look inside. It's not like anyone would know, or care.)`
				`	(Don't mess with it; delivering this cargo unmolested will earn the respect of your piratical peers.)`
					decline
			`	What could lay within? A cursory shake reveals no rattling, nor clinking. Probably not golden coins, then.`
			`	Rather, you find an old journal in a weatherproof plastic sleeve sitting atop a stack of old clothes. There doesn't seem to be any name anywhere on the journal, and most of the pages have been torn out.`
			choice
				`	(Peruse a few pages; perhaps you'll learn something.)`
				`	(Leave it alone. If you wanted to read things, you would've gotten an honest job.)`
					decline
			`October 3, '94`
			`	"Bastards took the farm. Funny how you go your whole life never seeing a man in a suit, and the first one you do is there to ruin everything. Well, fine. I'm strong. I'm smart. I can go to the city and find work."`

> 			`	The next several pages are filled with landscape sketches, scenes from a verdant farming world. Trees loom tall in the divisions between fields; flowers are abundant where nature hasn't been tamed by the colonist's hand. Here, a flock of sheep graze. There, a ship streaks across the sky, the rest of the drawing seemingly contorted around it as if the world itself is focusing on the vessel as it speeds into the unknown.`

Reviewer 4: ```suggestion
			`	The next several pages are filled with landscape sketches, scenes from a verdant farming world. Trees loom tall in the divisions between fields; flowers are abundant where nature hasn't been tamed by the colonists' hand. Here, a flock of sheep graze. There, a ship streaks across the sky, the rest of the drawing seemingly contorted around it as if the world itself is focusing on the vessel as it speeds into the unknown.`
```

			`November 15, '94`
			`	"Today's the day I'm born anew. Not with one of those cults, but with the "merchant" who comes by every season. He said there's room on his crew for a young man who's not afraid to get his hands dirty, and if I help around the ship, and he likes me by the time he takes off again, I can stay and take a share in their next haul... But I don't want to bring the law down on my family. I'll need a new name. "Voidmaster" sounds cool. I'll see if I can get the crew to call me that."`
			`	The remaining pages have been torn out.`
				accept
	on accept
		log `Found an old journal inside a chest from a ship I plundered. It had the following entries:`
		log `October 3, '94`
		log `	"Bastards took the farm. Funny how you go your whole life never seeing a man in a suit, and the first one you do wants everything you've got. Well, fine. I'm strong. I'm smart. I can go to the city and find work."`

> 		log `	The next several pages are filled with landscape sketches, scenes from a verdant farming world. Trees loom tall in the divisions between fields; flowers are abundant where nature hasn't been tamed by the colonist's hand. Here, a flock of sheep graze. There, a ship streaks across the sky, the rest of the drawing seemingly contorted around it as if the world itself is focusing on the vessel as it speeds into the unknown.`

Reviewer 4: ```suggestion
		log `	The next several pages are filled with landscape sketches, scenes from a verdant farming world. Trees loom tall in the divisions between fields; flowers are abundant where nature hasn't been tamed by the colonists' hand. Here, a flock of sheep graze. There, a ship streaks across the sky, the rest of the drawing seemingly contorted around it as if the world itself is focusing on the vessel as it speeds into the unknown.`
```

	choice
		`	(Proceed to Hangar 18.)`
	`	Just as promised, the datachip fits right into a slot in the wall, unsealing the airlock and allowing you into the hangar. Voices and light spill from within.`
	choice
		`	(Take a moment to listen.)`
		`	(Enter and announce yourself.)`
			goto CCOR
	# Commented lines to be uncommented when Unicode support is added
	#`	"Eu sei, mas é que eu realmente não acho que isso vai só acontecer. O vô do meu vô também passou a vida esperando a mesma coisa, e hoje eu e tu tamos aqui, negociando com capitalistas."`
	`	"Eu sei, mas e que eu realmente nao acho que isso vai so acontecer. O vo do meu vo tambem passou a vida esperando a mesma coisa, e hoje eu e tu tamos aqui, negociando com capitalistas."`
	#`	"Eu preferiria se tu não fosse tão negativo, camarada. Tu sabe tão bem quanto eu que é só uma questão de esperar as coisas entrarem em colapso. Porra, tem uma guerra civil arriscando começar a qualquer momento, se segura por só mais um pouquinho! Inclusive, eu escutei a pouco que – "`

> 	`	"Eu preferiria se tu nao fosse tao negativo, camarada. Tu sabe tao bem quanto eu que e so uma questao de esperar as coisas entrarem em colapso. Porra, tem uma guerra civil arriscando começar a qualquer momento, se segura por so mais um pouquinho! Inclusive, eu escutei a pouco que-"`

Author: It was written by a Brazilian.

	`	Without warning a figure steps in front of the doorway, silhouetted by strong lights pointed at the doorframe from within. "In or out, you're letting through a draft. We can conduct business inside."`
	choice
		`	(In.)`
		`	(Out.)`
			goto retreat
	`	With a smile and a nod, the figure gestures you inside. "Good, good. In you go."`
	label CCOR

> 	#`	The hangar is dominated by an orange-painted Hauler of older make looming behind everything else. Once you step beyond the glare of the floodlight pointed at the door you can make out some details. Those aren't crates stacked in a rough perimeter around the door, but sandbags. Behind one row a tripod with an old-style machine gun rests, "Mãe Dois" painted in white upon the housing. The gunner, a tanned young woman with a cigarette hanging jauntily from her lips, notices your look and offers a friendly enough wave. Like the other five people in the room, she's wearing a burgundy military uniform, with gleaming strips of brass where one might usually find decorating ribbons.`

Reviewer 4: ```suggestion
	#`	The hangar is dominated by an orange-painted Hauler of older making looming behind everything else. Once you step beyond the glare of the floodlight pointed at the door you can make out some details. Those aren't crates stacked in a rough perimeter around the door, but sandbags. Behind one row a tripod with an old-style machine gun rests, "Mãe Dois" painted in white upon the housing. The gunner, a tanned young woman with a cigarette hanging jauntily from her lips, notices your look and offers a friendly enough wave. Like the other five people in the room, she's wearing a burgundy military uniform, with gleaming strips of brass where one might usually find decorating ribbons.`
```
Not sure tbh.

Author: "Make" describes a particular model or type of ship. This is correct.

> 	`	The hangar is dominated by an orange-painted Hauler of older make looming behind everything else. Once you step beyond the glare of the floodlight pointed at the door, you can make out some details. Those aren't crates stacked in a rough perimeter around the door, but sandbags. Behind one row a tripod with an old-style machine gun rests, "Mae Dois" painted in white upon the housing. The gunner, a tanned young woman with a cigarette hanging jauntily from her lips, notices your look and offers a friendly enough wave. Like the other five people in the room, she's wearing a burgundy military uniform, with gleaming strips of brass where one might usually find decorating ribbons.`

Reviewer 4: ```suggestion
	`	The hangar is dominated by an orange-painted Hauler of older making looming behind everything else. Once you step beyond the glare of the floodlight pointed at the door, you can make out some details. Those aren't crates stacked in a rough perimeter around the door, but sandbags. Behind one row a tripod with an old-style machine gun rests, "Mae Dois" painted in white upon the housing. The gunner, a tanned young woman with a cigarette hanging jauntily from her lips, notices your look and offers a friendly enough wave. Like the other five people in the room, she's wearing a burgundy military uniform, with gleaming strips of brass where one might usually find decorating ribbons.`
```
Same here.

	`	"Since you had the key I'll assume you're here to do business," says the oldest of the six soldiers - a dark-haired man in his apparent fifties whose uniform practically gleams with medals - as he ushers you into a seat. He's the only one wearing a hat, which you can only assume means he's in charge.. "Coffee?" The brew appears dark, the scent rich and glorious as it cuts through the nicotine haze of the hangar.`
	choice
		`	"I would love some, thank you."`
		`	"Actually, I'm just here to do business."`
	`	"Of course. Now, let's get to it."`
	`	The officer consults his datapad, glancing briefly towards the now-sealed entrance to the room. "Says here you had the chip for the million-credit deal. You've got the money, I assume." Meanwhile, one of his compatriots approaches, steadily waving over you with a scanner of some sort. Everyone seems decidedly relaxed until it pings. Hands immediately drop to guns, cigarettes fall from lips, and tension spills into the air. "Escuta, quadril esquerdo," states the soldier with the scanner.`
	choice
		`	(Stay cool.)`
			goto cool
		`	(Go for your gun.)`

> 	`	You're promptly clobbered by the soldier standing behind you with the scanner. Rather than shoot, they'd simply opted for using what they had at hand - a heavy metallic sensor-bar. "Baita soco, camarada!" is the last thing you hear before blacking out.`

Reviewer 4: ```suggestion
	`	You're promptly clobbered by the soldier standing behind you with the scanner. Rather than shoot, they'd simply opted for using what they had at hand - a heavy metallic sensor bar. "Baita soco, camarada!" is the last thing you hear before blacking out.`
```

			`	Noon's sun beats down on you. Even with the chill that usually haunts this region of the planet at this time of year, it's enough to bring sweat to your brow. Silence fills the town square, and slavers shy back to leave only four things that matter in the area: You, the slaver boss, and your respective pistols.`
			`	A bird cries in the distance.`
			choice
				`	(Draw!)`
			`	The slaver boss is fast, but you're faster. You're able to slip your sidearm out of its holster, level it, and cleanly blow him away before he can so much as draw a bead on you - and so it's with the thrill of victory that you die, as you're unceremoniously cut down by massed fire from the gang of slavers surrounding your little duel.`
			`	The few colonists that are left behind after the slavers have filled their holds honor you with a short burial and a bottle of local moonshine poured out over your cairn in respect for the brave, but fairly ill-conceived, attempt to defend them.`
				die
			label attack
			`	You retreat to your ship, unable to help these poor souls...`

> 			`	... Or at least, that's what you want the slavers to think. They're caught unaware as you streak back towards the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to so much greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village back to the landing pad.`

Reviewer 4: Just a technicality, but it is possible for First Last's ship to be without weapons. Wouldn't make a very successful pirate though.

Author: I like this idea. How do we go about implementing it?

Reviewer 6: "deterrence" > 0 will make sure you have some gun

Reviewer 10: > I say make them die if they have 0 weapons.

A bit of a far out idea, but you could have them crash their ship into the pirates. Even with a sparrow I don't think that would be pleasant.

> 			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`

Reviewer 4: Same here.

					goto duel
				`	(Return to your ship. This is none of your business.)`
					goto retreat
				`	(Return to your ship... and start strafing the slavers.)`
					goto attack
			`	You're given an appraising look-over by the slaver chief. "... Yeah, you'd be too much trouble to sell. What'cha want, kid? We're busy here."`
			choice
				`	"These colonists are my property by right of conquest. If you're taking them, I expect to be fairly compensated."`
				`	"Actually, I was just leaving."`
					goto retreat
				`	"Whoah, whoah, whoah, I'm cool with forcing people to give me stuff because I'm holding them at gunpoint, but forcing them into slavery? That's sick."`
					goto challenge
			`	The chief works his jaw for a moment, deep in thought. "Alright. You soundly thrashed the last bunch of bastards who were here, so I'm inclined to think we're best off doing business. Wholesale rate on these skins is ten thousand a body. 'course, that's with no shipping and handling - factor that in and we can afford to give you ehhh... Quarter million for the lot of 'em. You'll still have some left once we're done, mind."`

> 			choice

Reviewer 3: I think there should be an option to refuse the offer here as being too low, if we're going to take the 'terrible person' angle with this option.

			`	A bird cries in the distance.`
			choice
				`	(Draw!)`
			`	The slaver boss is fast, but you're faster. You're able to slip your sidearm out of its holster, level it, and cleanly blow him away before he can so much as draw a bead on you - and so it's with the thrill of victory that you die, as you're unceremoniously cut down by massed fire from the gang of slavers surrounding your little duel.`
			`	The few colonists that are left behind after the slavers have filled their holds honor your sacrifice by rolling your body into a ditch, spitting on it, and then promptly forgetting you ever lived.`
				die
			label retreat
			`	You retreat to your ship, unable or unwilling to defend your extortion-subjects against a slaver incursion.`
				flee
			label challenge
			`	"Yeah? And what do you think you can do about that?"`
			choice
				`	"You raise a compelling point, sir, and I wish you a good day while I leave to think further upon that."`
					goto retreat
				`	"I think I can kick your ass, that's what."`
					goto duel

> 				`	(Make your excuses and return to your ship. You'll be back - once your engines and weapons are warmed up.)`

Reviewer 4: Also here.

Author: What are you pointing out in this block?

Reviewer 6: Same thing, your ship could be weaponless

			label attack

> 			`	You hustle back to your ship, slavers jeering at your back. They're not laughing long, however. They're caught unaware as you streak back toward the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to so much greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village and back to the landing pad.`

Reviewer 4: And here.

> 			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`

Reviewer 4: And here.

> mission "You Are A Pirate"

Author: Yeah, they both have the same outcome - one's a "I need to prove myself" story, the other is a "I've been living the life and embrace officially becoming what I already am" story.

	landing
	source
		government "Pirate"
	to offer
		not "chosen sides"
		or	
			and
				or
					"reputation: Republic" < -10
					"reputation: Syndicate" < -10
					"reputation: Free Worlds" < -10
				"reputation: Merchant" < -10
			"reputation: Pirate" > 20
	on offer
		conversation "You Are A Pirate"
	on accept
		event "became a pirate"

> 		fail

Reviewer 7: Yep, good call, thought it was invisible all this time.

conversation "You Are A Pirate"
	`Preparing to land is second nature by now: patch any perforations in the inner hull, get in touch with ground control, and then make sure your chosen parking pad isn't controlled by extortionists. When you disembark it's with a pistol at your hip, bribe money in your pocket, and a swagger in your step; to leave behind any one of these is to invite trouble. Perhaps a life of violence on the high skies and association with the rougher elements of the galaxy weren't what you had in mind when you left your homeworld, but it's what you found.`
	choice
		`	(This is my life now. Piracy, smuggling, the existence of the outlaw. I'm ready to embrace it.)`
		`	(This isn't what I wanted. This isn't the life I imagined when I decided to take to the high skies.)`
			decline

> 	`	Resolve will carry you far in this world. Tonight it's carried you to a spaceport paint supplier; you return to your vessel with a pallet loaded with blacks and reds. Apply it to your vessel, make your intentions known to the galaxy. There can be no pretense of peace when you fly girded in blood and midnight, but maybe that's exactly the way you want it to be.`

Author: If the player is reading this they've already made up their mind, no need to render it in subjunctive - and it would ruin the flow.

I do like the credit cost idea, though.

> 	`	Resolve will carry you far in this world. Tonight it's carried you to a spaceport paint supplier; you return to your vessel with a pallet loaded with blacks and reds. Apply it to your vessel, make your intentions known to the galaxy. There can be no pretense of peace when you fly girded in blood and midnight, but maybe that's exactly the way you want it to be.`

Reviewer 11: I don't think it ruins the flow myself - the following sentence includes a hypothetical anyways. But up to you!

	choice
		`	(Don the smock and mask, turn on the compressor, conceal your old life beneath bellicose black.)`
		`	(No. Not now, perhaps not ever. The galaxy at large may consider you a pirate, but you're not ready to adopt their colors.)`
			decline

	`	Some paint covers all manner of sin. Some speaks of the intention to commit yet more. You've applied the latter sort to your ship.`
		accept

mission "I Want To Be A Pirate"
	clearance
	name "Become a pirate"
	description "Being a pirate in the eyes of the law isn't the same as being one in the eyes of fellow buccaneers. You'll have them know your name - but first you must make one for yourself."
	source
		government "Pirate"
	destination "Greenrock"

> 	to offer

Reviewer 7: The first mission is if you already grinded out the reputation, already earning the respect. You effectively are already one in that case, so the landing mission just formalizes it and grants you the swizzle. This second mission is for players coming in more as a merchant, but who aspire to be a pirate, looking to get into the pirate life by doing specific favors.

> 	to offer

Author: Yeah, this one's for greenhorns who've done absolutely nothing wrong (yet) but - for one reason or another - have decided that it's a pirate's life for them.

> 	to offer

Reviewer: I guess such an idea seems odd to me. "Hello, I'd like to be a pirate, but I'm still on the right side of the law as I haven't actually done anything bad yet!" seems like an unlikely thing to ever happen.

> 	to offer

Reviewer 7: Grinding up rep in Endless Sky is a pretty tedious and nebulous task. I would know, I created a pirate file and spent an hour bribing pirates, merchants, and militia in Aldhibain to slowly repair my way into being a pilot. The upper mission is the acknowledgement that someone put in the work, and is more of an easter egg/shortcut for those people. For people coming into the game on newer pilots, looking to be a pirate, this is more of the standard on ramp, offered by just landing on a pirate planet.

> 	to offer

Author: It is indeed odd, which is why the NPCs react as they do - the player in this is clearly a starry-eyed moonbrain, and they regard your request with confusion and amusement but entertain it nonetheless. 

Yeah. Nobody *wants* to be a pirate. But sometimes people have bigger dreams than sense, and they're the ones that tend to get big things done - if only through dumb luck.

All that aside, it's a Secret of Monkey Island reference.

> 	to offer

Author: To put it another way, you're a young person from the middle of nowhere with no inheritance and no education, just a dream and a pilot's license.

You won't make it big by working for some company, they tend to want a degree.

You can't just buy your way to the top (yet).

But organized crime? That's a traditional route for people who have nothing more than ambition to rise to heights otherwise inaccessible.

> 	to offer

Reviewer 7: It's also meant to lean into the Sparrow's shipyard description, and all those other pirate interceptors you see out in the wild. There needs to be a *relatively* easy path to becoming a pirate, otherwise, how'd all those tiny ships you massacre in the hundreds or thousands get to that point?

> 	to offer

Reviewer: All right!

	`	"Mark my words, there are better things in life than simply seeing your enemies laid to waste. You want them to respect your power, but not feel so threatened that they would devote themselves to your destruction. When you kill, do so thoughtfully." The Mercenary, meanwhile, is sporting thoroughly flushed cheeks by this point - though she yanks her hand back and gives the Capo a gentle smack on the jaw before averting her gaze. There's a hint of a smile upon her chapped lips, and regret hidden deep within tired eyes.`
	choice
		`	"I don't understand. Could you explain that more plainly?"`
		`	"Tell me what I need to do."`
			goto explanation

> 	`	The Capo groans, shakes his head mournfully, and leans back to excuse himself from the conversation for a minute. The Mercenary quickly fills the conversational. "Look, it's simple. You shoot people, they shoot back. You knuckle under at every opportunity, they keep shooting because they think you're weak. If you want to be left alone you need to prove that you're not worth the effort it would take to plunder you, yeah? That's... Well, it's what these little tasks we've mentioned are about. You need to establish a reputation. This is how you do it - and once you've done that, other pirates should leave you alone owing to either fear or respect."`

Author: Yeah, I have no idea how I managed to write that garbage sentence - what the hell? Must've been distracted.

> 	`	She leans back in her chair, matching gaze with the Capo. "And you, don't push it. Not here, you know what my boss thinks." Wistfulness drips from her words, and her gaze turns elsewhere; this person's thoughts clearly lie elsewhere at the moment.`

Author: Much better, thank you.

	`	"Now-" the Corsair begins. "I know a juicy prize is going to be sailing-" The other two turn to fix the Corsair with a skeptical gaze. The Capo's maw slips wide for a moment, as if he wants to interrupt, but eventually it closes silently once more. "-SAILING, I say, through Umbral. In fact, any buccaneer worth their salt knows she'll be there, so you're liable to have more than a little competition. Board her, plunder her, keep what ye like - but should ye find an antique wood lockbox, yer' to bring it back to me without peeking within. That'll be my price for spreading the word of yer' glorious ascent to piratehood.`
	`	"As for me?" The Devil's Due member takes her turn to speak. "Take a million credits to Smuggler's Den in Men. I don't care where you get the money. In fact, I don't want to know. Just drop by with that million, complete a little transaction I'll have arranged by then, and I'll get you what you want. Prestige. Respect. Fame. Infamy, whatever." A winning smile is flashed your way, and your hand is plucked up, shaken; a datachip is pressed into it. "Contact for an associate of mine, on the station I mentioned. Take the money there, bring the item back to me."`

> 	`	Finally, the Capo takes his turn. "Have you heard of the planet Darkstone, in the system Han? Pathetic bunch of dirt-farmers, only worthy of enslavement. Impose yourself as their master, take the meager tribute they offer, and return. Know that you'll make enemies this way, but really - if you want to be a pirate, I'm sure that's not something that concerns you overmuch."`

Author: There needs to be a close quote here.

		goto tributeskip
	label defender
	choice
		`	(You address the Capo.) "I conquered Darkstone, defeating the gang that once claimed it. When others came to enslave my subjects I defeated them as well."`
	`	"You little shit. I gave you a job to do, an opportunity, and you repay me by killing my men? Did you think you could just muscle in to the trade without people backing you up?" Cold rage burns within the cyborg's eyes, his expression hard to read given lack of lips and most facial muscles.`
	choice
		`	"They got what was coming to them. That territory was mine by right of conquest, and by trespassing they signed their own death warrants. You respect my turf and I'll respect you, understood?"`
			goto grudgingrespect
		`	"I didn't know those were your men, I simply saw raiders coming in to steal from my new tributaries and I didn't want that to happen. I'm sorry."`
			goto weak
		`	"Damn slavers. If I'd known that's what you'd intended all along I never would have gone along with your plan. You disgust me."`

> 	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`

Author: It was either this or apply a condition in the lead-up to line 172. The followup lines to this blurb are necessarily different because they come from different start conditions.

> 	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`

Reviewer: I understand. I'd slightly prefer a condition and a common label.

> 	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`

Author: I don't think I can give these a common label and still have the differences be within the same paragraph, you can only do labels after a new line.

> 	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`

Author: Okay, looking this over I could probably make a weave of applying conditions and then labels that redirect once you have the right condition, but that's massively more complicated than just having two different flows of conversation which happen to include three quarters of a paragraph in common.

> 	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`

Author: (That is to say, I'd just very much rather not and leave it as-is)

> 	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`

Reviewer 7: I definitely get wanted the text to be as compact as possible. But yeah, these are not the same line and can't be shared between dialog trees without throwing an ugly line break in there.

> 	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`

Reviewer: ok.

	`	"But I can see there can be no mutually beneficial arrangements between us. You did the job I told you to, true enough, but I'd never sully my reputation by vouching for someone so... stupidly idealistic. Maybe these other idiots will have some use for you." A moment passes before the Capo gives the Mercenary's knee a squeeze. "Not you, dear. I meant the other guy." He's given a roll of the eyes for his trouble.`
		goto defenderspeech
	label grudgingrespect
	`	The Capo stares you down, sizing you up, waiting for some sign of weakness on your part. None is given. Finally, his eyebrows relax and his face seems, accordingly, somewhat less hostile.`

> 	`	"Fine. Yes. You speek the truth, you do - you had no way of knowing those slavers were in my employ, and you did well to defend your territory. Truthfully, I didn't expect you'd be able to stand up to my boys, or even fight off the existing occupants. The fault is mine for underestimating your abilities."`

Author: Damn, this one's embarassing.

		goto pirate
	label smugglingalt
	choice
		`	"I've got your million-credit... thing, and a splitting headache."`
	`	"I'll be blunt." The Mercenary thoughtfully presses her fingertips together, as if that would somehow magically make whatever words come next more diplomatic.`
	`	"You're the most empty-headed spacer I've seen that isn't yet a corpse. Next time you want to try a stunt like that, do us all a favor and just take a walk out the airlock first. I've met asteroids with more sense than you. Drawing a gun on my contacts after they found a bug on you? You've got some wild ideas about your own capabilities, but... Somehow you were able to talk your way out of it, so perhaps you're not a lost cause after all. Whatever the case, I can't have a reputation as one who goes back on a deal, so I'll hold up my part of the bargain. Now, the canister please."`
	choice
		`	(Hand over the canister.)`
	`	The merc gingerly takes the hazard-warning emblazoned canister from you, as if half expecting you to drop it on the floor and release who-knows-what into the room. The other two at the table visibly exhale in relief once the canister's out of your hands.`
	label pirate

> 	`	"And we've got one more thing for you," the Corsair comments before pulling a small booklet out of his coat. It's slid across the table before you. It appears you've been given some sort of coupon book for an establishment called "SLEEZY STEEV'S SCRAP AND SURPLUS". Most of the coupons have been torn out, but one of the remaining offers apparently entitles you to a hefty discount on black and red paint.`

Author: I'm mostly willing to gloss over it for the sake of narrative smoothness. The player character is caught up in the excitement of becoming a pirate; disrupting that flow for the sake of a little bit of maintaining the simulation is a worthwhile price in my appraisal.

	`	"This is what you came for," the officer explains. "If it's to your satisfaction you can leave with it, and we'll keep that credit chip."`
	choice
		`	(Inspect the object.)`
	`	It's a steel canister about the size of a thermos. The brilliant yellow appearance from a distance proves to be thanks to a plethora of warning labels stuck all over the canister. An unsettling quantity of stylized skulls and exclamation points are visible, leading to an almost polka-dotted appearance if you squint. Present in the middle of the warning labels is a simple placard welded to the canister. It reads, plainly: "Drought-Resistant Seeds". No other documentation as to what's actually in the can seems to be forthcoming.`
	choice
		`	"Looks good to me, thank you."`
			goto complete
		`	"What in the void is this?"`
	`	The dark-haired officer simply smiles gently. "It is what it is. Now take it and run along to your employer. We're done here."`
	branch complete
		not "clobbered"
	`	The soldiers give you a moment to dress yourself and gather your things back up, though the surly looks you're fixed with make it clear they're keen to see you go.`
	label complete
	action
		payment -1000000

> 	`	The transaction is complete. You've spent a million credits out of your own pocket for something that needs to be delivered to a mysterious mercenary you met in a stinking bar. This is definitely the path of wisdom that you walk. Thankfully, no further obstacles ambush you as you return to your ship.`

Reviewer 11: `This is definitely the path of wisdom that you walk.`
Banger of a line. Made me chuckle.

		accept
	label retreat

> 	`	You make your apologies - wrong hangar - and beat a hasty retreat. The door seals behind you. On the way to your ship you pass the janitor once more, his ear firmly pressed to the handle of his mop. Upon seeing you he becomes visibly startled and drops his tools all over the floor. What a strange fellow.`

Reviewer 11: (This did too)

			has "hero of Darkstone"

mission "Pirate Intro: Domination Check-in"
	clearance
	invisible
	landing
	source "Darkstone"
	to complete
		has "event: slaver encounter"
	to fail
		not "Pirate Intro: Domination: active"
	to offer
		has "Pirate Intro: Domination: active"
		has "tribute: Darkstone"
	on offer
		event "slaver encounter" 2
		conversation
			`Apparently the mess you made in orbit was visible even from the ground, as when you come in to land there's a delegation of frankly depressed-looking colonists waiting to greet you. The only member of the party who looks even vaguely chipper is a goat that they're holding on the end of a rope, who's otherwise merrily grazing on whatever foliage clings to the edge of the packed-dirt landing pad.`
			choice
				`	(Wait for the colonists to speak first.)`
					goto silence
				`	"You lot know how it works. Old master's dead, that makes me the new one. Pay up and I'll be off."`
					goto tribute
				`	"Kneel before me, peasants!"`
					goto silence
				`	"Hello! I'm fairly new to this whole 'conquering the galaxy' thing, but I think that beating that crew of scoundrels like I did makes me the new boss of this place, right?"`

> 			`	The colonists exchange looks. They seem far too lost in their own misery to even try dealing with your obnoxiously chipper demeanor, and you're simply given a chorus of shrugs and murmured 'Yah, sure, whynot' responses.`
> 			`	"Whatever you say, chief."`

Reviewer 11: ```suggestion
			`	The colonists exchange looks. They seem far too lost in their own misery to even try dealing with your obnoxiously chipper demeanor, and you're simply given a chorus of shrugs and murmured "Yah, sure, whynot" responses. "Whatever you say, chief," one mutters.`
```
Having that on a line by itself made it confusing who exactly was speaking to me.

			choice
				`	"These people are my responsibility, and I'll fight you for their freedom."`
				`	(Return to your ship. This is none of your business.)`
					goto retreat
				`	(Return to your ship... and start strafing the slavers.)`
					goto attack	
			label duel
			`	The leader of the slavers turns to face you, attention finally pulled from his task. "You've got some nerve picking a fight with us, kid. You really think these poor bastards are worth dying for?"`
			`	Noon's sun beats down on you. Even with the chill that usually haunts this region of the planet at this time of year, it's enough to bring sweat to your brow. Silence fills the town square, and slavers shy back to leave only four things that matter in the area: You, the slaver boss, and your respective pistols.`
			`	A bird cries in the distance.`
			choice
				`	(Draw!)`
			`	The slaver boss is fast, but you're faster. You're able to slip your sidearm out of its holster, level it, and cleanly blow him away before he can so much as draw a bead on you - and so it's with the thrill of victory that you die, as you're unceremoniously cut down by massed fire from the gang of slavers surrounding your little duel.`

> 			`	The few colonists that are left behind after the slavers have filled their holds honor you with a short burial and a bottle of local moonshine poured out over your cairn in respect for the brave, but fairly ill-conceived attempt to defend them.`

Reviewer 11: ```suggestion
			`	The few colonists that are left behind after the slavers have filled their holds honor you with a short burial and a bottle of local moonshine poured out over your cairn in respect for the brave, but fairly ill-conceived, attempt to defend them.`
```

				die
			label attack
			`	You retreat to your ship, unable to help these poor souls...`

> 			`	... Or at least, that's what you want the slavers to think. They're caught unaware as you streak back towards the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to so much greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village back to the landing pad.`

Reviewer 11: ```suggestion
			`	... Or at least, that's what you want the slavers to think. They're caught unaware as you streak back towards the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village back to the landing pad.`
```

Author: Keeping it this way.

> 			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought on-line, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`

Reviewer 11: ```suggestion
			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`
```
Any reason you used `on-line`?

			`	A bird cries in the distance.`
			choice
				`	(Draw!)`
			`	The slaver boss is fast, but you're faster. You're able to slip your sidearm out of its holster, level it, and cleanly blow him away before he can so much as draw a bead on you - and so it's with the thrill of victory that you die, as you're unceremoniously cut down by massed fire from the gang of slavers surrounding your little duel.`
			`	The few colonists that are left behind after the slavers have filled their holds honor your sacrifice by rolling your body into a ditch, spitting on it, and then promptly forgetting you ever lived.`
				die
			label retreat
			`	You retreat to your ship, unable or unwilling to defend your extortion-subjects against a slaver incursion.`
				flee
			label challenge
			`	"Yeah? And what do you think you can do about that?"`
			choice
				`	"You raise a compelling point, sir, and I wish you a good day while I leave to think further upon that."`
					goto retreat
				`	"I think I can kick your ass, that's what."`
					goto duel
				`	(Make your excuses and return to your ship. You'll be back - once your engines and weapons are warmed up.)`
			label attack

> 			`	You hustle back to your ship, slavers jeering at your back. They're not laughing long, however. They're caught unaware as you streak back towards the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to so much greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village and back to the landing pad.`

Reviewer 11: ```suggestion
			`	You hustle back to your ship, slavers jeering at your back. They're not laughing long, however. They're caught unaware as you streak back toward the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village and back to the landing pad.`
```

> 			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought on-line, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`

Reviewer 11: ```suggestion
			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`
```

# by waiving all of his or her rights to the work worldwide under copyright law, including all
# related and neighboring rights, to the extent allowed by law.
#
# You can copy, modify, distribute and perform the work, even for commercial purposes, all
# without asking permission.
#
# In no way are the patent or trademark rights of any person affected by CC0, nor are the
# rights that other persons may have in the work or in how the work is used, such as publicity
# or privacy rights.
#
# Unless expressly stated otherwise, the person who associated a work with this deed makes no
# warranties about the work, and disclaims liability for all uses of the work, to the fullest
# extent permitted by applicable law.
#
# When using or citing the work, you should not imply endorsement by the author or the affirmer.

mission "You Are A Pirate"
	landing
	invisible
	source
		government "Pirate"
	to offer
		not "chosen sides"
		or
			and
				or
					"reputation: Republic" < -10
					"reputation: Syndicate" < -10
					"reputation: Free Worlds" < -10
				"reputation: Merchant" < -10
			"reputation: Pirate" > 20
	on offer
		conversation "You Are A Pirate"
	on accept
		event "became a pirate"

> 		fail

Reviewer 12: We don't need to trigger this event in the on accept that immediately fails the mission anymore. We can instead use a conversation action to trigger the event after you've made the conversation choice that would lead to accepting the mission currently. We'd then always decline the mission.

> conversation "You Are A Pirate"

Reviewer 12: Why is this defined as a root conversation when it's only used in one mission?

Reviewer 7: Was originally a case of ease-of-editing and not being sure if it'd show up in multiple missions or not. Can have that integrated now, though.

	choice
		`	(Conceal my old life beneath bellicose black.)`
		`	(No. The galaxy at large may consider me a pirate, but I'm not ready to adopt their colors.)`
			decline
	`	You don the smock and mask, turn on the compressor. Some paint covers all manner of sin. Some speaks of the intention to commit yet more. You apply the latter sort to your ship.`
		accept

mission "I Want To Be A Pirate"
	clearance
	invisible
	source
		government "Pirate"
	destination "Greenrock"
	to offer
		not "You Are A Pirate: offered"
		not "chosen sides"
		or
			"combat rating" > 150
			has "ships: Interceptor"
			has "ships: Drone"
			has "ships: Fighter"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
	on offer
		conversation "I Want To Be A Pirate"
	to complete
		or
			has "Pirate Intro: Boarding: done"
			has "Pirate Intro: Wealth: done"
			has "Pirate Intro: Domination: done"
	on complete
		event "became a pirate"
		conversation "I Want To Be A Pirate Conclusion"

> conversation "I Want To Be A Pirate"

Reviewer 12: Why are various conversations defined as root nodes instead of in the missions themselves? They look to only be used once each.

> conversation "I Want To Be A Pirate"

Reviewer 12: This one is a very lengthy conversation. We generally want to avoid very long conversations. Something MZ has said is that this is a video game, not a novel. Is there anything here we could cut down, or is it necessary that the conversation have this much meat to it?

	`	"Aye, it be not enough to simply cover yer' ship with the old black-and-red. A pirate must earn the respect - or fear - of their fellows, lest they be seen as fair plunder," the Corsair adds. This is followed up with an entirely gratuitous, but quite hearty, "Arrrrgh!"`
	`	The Capo speaks last, jaw motionless; his voice is provided by a cybernetic vocoder, producing a honeyed if gently synthetic sound. "Indeed. There's three things you'll need to do before you'll be taken seriously. Speaking personally, I'm always on the eye for upcoming talent. Prove that you've got what it takes and I'm sure at least one of us would make it known within our circles that you aren't to be trifled with - and perhaps even worth offering work to. But first, those tasks."`
	choice
		`	"Does that guy always speak like that? With the 'arrrgh's and such?"`
		`	"Oh, I know! I need to master theft, swordplay, and treasure hunting!"`

> 			goto hahareferencegoeshere

Reviewer 12: Sensible label names, please.

	choice
		`	"What trade is that, exactly?"`
		`	"Thank you. I've conquered a world, but I'm only getting started."`
			goto outlawapproval
	`	"Human resources, naturally." The Capo lets out a dark laugh, eyes thin and aspect sharklike as he fixes you with a predatory grin.`
	choice
		`	(Remain impassive, not caring one way or another about the fates of others than yourself - slavery is neither your business nor your problem.)`
			goto outlawapproval
		`	(Remain silent, though your face will betray your disgust.)`
			goto slaveryspeech
		`	"Of course. Slavery. A fine and upstanding industry. Someone has to do all the work, and it's damn well not going to be me."`
		`	"Slavery? You're a monster."`
			goto slaveryspeech
	action
		set terribleperson
	`	"I'm glad we see eye to eye. Protection racket, landlord, slavemaster; whatever people want to call it, it's simply the natural order of things that some are strong, and some are weak. The strong thrive because they put to better use that which the weak produce. And you? I know which of the two you are now. I'll put your name into the ears of others like us."`

> 	# The ring token is for integration into the pirate campaign

Reviewer 13: ```suggestion
	# The ring token is for integration into the pirate campaign.
```

	action
		set RookieSlaver
	`	"So I heard, so I heard! It was my boys you were selling those dirt-grubbers off to. Mind you, I wasn't expecting to need to cut you in on this, but you did chase off the old owners so I suppose it's fair. One thing is sure: you've got a winner's instincts. I'm proud to sponsor you, and rest assured... people will know you're a cutthroat bastard, ready to sell any body that falls into your hands. Just like me." The Capo pauses to wipe a tear from his eye. "I swear, if I still had lips I'd be smiling."`
	label outlawapproval
	`	"Now, I'll do as I said and help make your name known to those who matter, and the lowlifes that don't. Pirates will respect you for the deeds you've done."`
		goto tributeskip
	# label ring
	# `	The Capo extends his palm, a shining silver ring glistening in the center. Unadorned by any manner of stone, it's a simple, subdued piece of jewelry. Upon the front a crest has been stamped. "A token of my approval. Consider it sponsorship, and my trust that you'll do what need be done. Take it to Gienah. People will know what it means, and you'll find lucrative work that lines up nicely with the skillset you've demonstrated."`

> 	# `	The cyborg offers a genial smile - or at least as much as one can without lips. It's uncanny how expressive articulated artificial jaws can be these days.`

Reviewer 12: Avoid having commented out stuff like this. I understand that this is still a draft, but we eventually want almost none of this in the final version.

	label tributeskip
	branch boardingskip
		not "Pirate Intro: Boarding: done"
	choice

> 		`	(Slap the top of the Corsair's trunk.) "I've brought the antique lockbox and defeated the other pirates who came for it."`

Reviewer 12: Avoid combining actions and dialog in the same choice. If you want a choice to entail the player doing an action, then have the choice just be the action and then have the resulting lines contain the player's dialog.

Reviewer 7: Do we have any instances of that in the base game? My memory is any non-choice instances of <first> <last> dialog are in response to other NPC dialog, and that have a <first> <last> action followed with a newline and then their dialog might even be more clunky. Can make that change if necessary, but might be worth keeping.

			`	Completing any of these truly important quests and reporting back to the ne'er-do-wells on Greenrock will make you a pirate.`
				decline

mission "Pirate Intro: Boarding"
	landing
	name "Board a merchant"
	description `Board a merchant.`
	destination "Greenrock"
	waypoint "Umbral"
	to offer
		has "I Want To Be A Pirate: active"
	to fail
		has "I Want To Be A Pirate: done"
	on offer
		event "add merchant targets"
	to complete
		has "Plunder Defense: done"

fleet "Merchant Targets"
	government "Merchant (Target)"
	names "civilian"
	cargo 1
	personality
		heroic
	variant 1
		"Freighter"

government "Merchant (Target)"
	swizzle 5
	
	"player reputation" 10
	"attitude toward"
		"Pirate" -.2
		"Korath" -.2
	"bribe" .05
	"fine" 0
	"friendly hail" "friendly civilian"
	"hostile hail" "hostile civilian"

mission "Plunder Defense"
	name "Return with spoils"
	description `Defeat your rival pirates in Umbral, then return to <destination> with the spoils from the <origin> to confirm your loyalty.`
	boarding
	source

> 		government "Merchant (Target)"

Reviewer 12: Ditch the mission-specific government and instead have this occur on any merchant, but you may want to select a few specific systems in which this interaction can occur.

Reviewer 7: Part of Umbral was to have an out of the way location where those merchants are up to something, then bringing the fight for the plunder between you and the pirates. The natural spawn rates are a bit too low for normal merchants, and other nearby uninhabited systems have much higher traffic from either merchants, militia/FW/Republic, or both. 

Could use a mission NPC, which would hover around too much _or_ be too hard to nail down, or add in the fleet definition anyways for those dead ends which is basically what's currently in use. I do like using a more elegant solution on paper in terms of a broader boarding mission, but I don't think there's a good fit with what's naturally in the game. The extra fleet event could stay with just the normal Merchant government, but I suspect future pirate jobs/missions could still make use of such a government, as well. If it has to be natural-spawning merchants, what system(s) should be used then?

	action
		set "clobbered"
	choice
		`	(What a headache.)`
	choice
		`	(The taste of blood fills your mouth.)`
	choice
		`	(I should open my eyes.)`
	`	You awaken tied to a chair, a headache pounding in your ears, and where there used to be a serving table with coffee your belongings are instead laid out.`
	`	There's your gun, your shirt - now bloodstained from the solid whack you'd sustained to the back of the head - your comms, your keys, and a credit chip valued at one million. Laid right in the middle of them is a minuscule device that you don't recognize - possibly due to the fact that it's been smashed into enough pieces to rival the starscape outside.`
	`	At least your captors were kind enough to shove you back into skivvies after stripping you down and emptying your pockets.`
	choice
		`	(Stay silent.)`
			goto interrogation
		`	(Offer undying curses to the bastards what did this to you.)`
		`	(Inquire as to what in the blazes is going on.)`

> 	`	You open your mouth, emitting several unintelligble croaks and an undignified cough. As one, the assembled soldiers turn to face you.`

Reviewer 3: ```suggestion
	`	You open your mouth, emitting several unintelligible croaks and an undignified cough. As one, the assembled soldiers turn to face you.`
```

		goto talking
	label better
	`	Precious. Delicious. Refreshing. The iron taste of blood is washed down your throat, and your headache slightly lessens. The pain is still there, but as far as you can tell you're not quite as dead as you felt upon waking.`
	label talking
	`	The officer settles into the chair across from you. "You brought the money, so I'm inclined to believe you weren't simply here to sell us out. Given your... decisions today, I'm also inclined to believe you may also just be an idiot. More importantly, you brought this bug we found on you. Care to explain yourself? If I like what you say, you may remain on this side of the airlock."`
		goto explanation
	label cool
	`	You keep your hands where they are and wait for the officer to speak. "There's a listening device on your left hip. Pocket, perhaps. I want you to slowly withdraw it and place it on the table in front of us. Gabriel, get the coffee out of here." A soldier hastily scoops up the beverage and disappears into the gloom of the hangar.`
	choice

> 		`	(Check your pocket.)`

Reviewer 12: Choice actions are 1st person.
```suggestion
		`	(Check my pocket.)`
```

			choice
				`	"You can keep the goat. I'll be back in a few days to collect."`
					accept
				`	"That's... pathetic. I expected more. I'm taking your goat, and I expect you sorry bunch of mud-scrapers to have some proper loot by the time I'm back."`
					accept
				`	"You know, I don't think this whole 'conquering tyrant' thing is going to work out from me. Can we just forget the whole tribute debacle? I'll leave you all alone. Sorry for the trouble."`
			`	"... You're messing with us, right?" Only a few of the colonists have the barest hints of hope visible on their faces. "That's... Well, it would be mighty fine to actually be able to keep something for a change. Alright!"`
			`	You can see the barest clue of a smile on the young man's lips, and light in his eyes that weren't there when you'd arrived. "And hey! You should come back in a couple days anyways, we don't have much but we'd like to throw you some sort of party to show our thanks. We make a mean corn liquor in these parts, and, well... We really hated that last crew that was camped out here. It'd mean a lot to us if you came by."`

> 			`(To release the planet from tribute, select it from orbit and use the hail key.)`

Reviewer 12: I like to avoid adding "gameplay tips" to conversations wherever possible. If this mission is changed to not use tributing, then we don't need this tip.
